src/gui/properties/trackerlistwidget.cpp

```diff
@@ -343,6 +344,28 @@ void TrackerListWidget::loadTrackers()
 
     loadStickyItems(torrent);
 
+    const auto toString = [](BitTorrent::TrackerStatus status) {
+        switch (status) {
+        case BitTorrent::TrackerStatus::Working:
+        case BitTorrent::TrackerStatus::WorkingWarning:
+            return tr("Working");
+        case BitTorrent::TrackerStatus::Updating:
+            return tr("Updating...");
+        case BitTorrent::TrackerStatus::NotWorking:
+            return tr("Not working");
+        case BitTorrent::TrackerStatus::NotContacted:
+            return tr("Not contacted yet");
+        }
+        return tr("Invalid status!");
+    };
+    const auto numberIf = [](auto val) {
+        return (val > -1) ? QString::number(val) : tr("N/A");
+    };
+    const auto setAlignment = [](QTreeWidgetItem *item) {
+        for (auto col : {COL_TIER, COL_PEERS, COL_SEEDS, COL_LEECHES, COL_DOWNLOADED})
```

Comment:
I think you should just write the concrete type out (`TrackerListColumn col`), to be more readable.